generator.go

```diff
@@ -285,7 +284,6 @@ func (g *Generator) Generate() ([]byte, error) {
 
 	var typeElements []*element
 	if g.namedTypes {
-		options.namedTypes = maps.Clone(g.typeElements)
```

Comment:
How does it now mutate g more than before? We are creating a new map with make. Clone only does a shallow copy.

Note that this has no effect here anyway. My previous commit changes it so we use make instead of clone, I simply forgot to delete this line.